services/navet-ms/mock/data.json

```diff
@@ -0,0 +1,5 @@
+{
+  "detail": {
+    "personalNumber": "195809262743"
```

Comment:
personalNumber should be an int/number?

Reply:
personalNumber changed to integer data type